src/core/Akka/Serialization/NewtonSoftJsonSerializer.cs

```diff
@@ -54,10 +89,38 @@ public NewtonSoftJsonSerializer(ExtendedActorSystem system)
                 ObjectCreationHandling = ObjectCreationHandling.Replace, //important: if reuse, the serializer will overwrite properties in default references, e.g. Props.DefaultDeploy or Props.noArgs
                 ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
                 TypeNameHandling = TypeNameHandling.All,
-                ContractResolver = new AkkaContractResolver(),
+                ContractResolver = new AkkaContractResolver()
+            };
+
+            _serializer = JsonSerializer.Create(_settings);
+        }
+
+        public NewtonSoftJsonSerializer(ExtendedActorSystem system, Config config)
+            : base(system)
+        {
+            var serializerSettings = NewtonSoftJsonSerializerSettings.Create(config);
```

Comment:
Also: what are the default JSON.NET HOCON settings? Shouldn't these be expressed somewhere also?